content/blog/svglite-2-0-0/index.Rmd

```diff
@@ -0,0 +1,258 @@
+This change means that svglite now have direct access to all your installed fonts along with whatever you have registered with `register_font()` and `register_variant()`.
+If you have registered any OpenType features or alternative weights with a font you use these will now be encoded into the CSS of the text block:
+
+```{r, results='hide'}
+library(systemfonts)
+register_variant(
+  name = "Montserrat Extreme", 
+  family = "Montserrat", 
+  weight = "semibold",
+  features = font_feature(ligatures = "discretionary", letters = "stylistic")
+)
+
+p <- ggplot() + 
+  geom_text(
+    aes(x = 0, y = 1, label = "This text should definitely differ"),
+    family = "Montserrat",
+    size = 6
+  ) + 
+  geom_text(
+    aes(x = 0, y = 0, label = "This text should definitely differ"),
+    family = "Montserrat Extreme",
+    size = 6
+  ) + 
+  theme_void() + 
+  expand_limits(y = c(-1, 2))
+
+svg <- svgstring(fix_text_size = FALSE, standalone = FALSE)
+plot(p)
+invisible(dev.off())
+
+svg()
+```
+
+``` {.svg}
+`r svg()`
+```
+
+If you compare the two `<text>` tags in the end you can see that the last one has the equivalent of *semibold* (600) encoded as the weight, along with the registered font feature settings (*dlig* is the tag for discretionary ligatures and *salt* is the tag for stylistic alternatives).
+
+The switch to using systemfonts means that two old arguments to svglite: `system_fonts` and `user_fonts` have become superseded by the font registration in systemfonts.
+While they work for now they will likely become deprecated in the future as we streamline the text handling among our graphics devices.
+
+The last addition to font handling in svglite is the support for embedding web font imports in the output.
+As we discussed in the [editability] section the output from svglite will only render correctly if the used fonts are available.
+This is not a given when using non-standard fonts, but CSS provides a safety net by allowing the viewer to download fonts on the fly if required.
+You can specify a web font in two ways:
+
+1.  As a URL (e.g. from Google Fonts) which will be used in an `@import` directive.
+
+2.  Using `font_face()` which allows you to manually set the location of different font files and local backup fonts along with a identification name and potential styles to use.
+    This will all be collected into a `@font-face` directive.
+
+We can use this with the example above to make sure that the SVG renders correctly even on computers that doesn't have the Montserrat font installed.
+
+```{r, results='asis'}
+svg <- svgstring(
+  width = 6, height = 2,
+  web_fonts = list(
+    "https://brick.freetls.fastly.net/Montserrat:400,600"
+  ),
+  fix_text_size = FALSE, 
+  standalone = FALSE
+)
+plot(p)
+invisible(dev.off())
+
+svg()
+```
+
+I'm using [Brick](https://brick.im) as a web font host since they include all OpenType features in their served font (which we need here), but the mechanism is the same for using e.g.
```

Comment:
```suggestion
I'm using [Brick](https://brick.im) as a web font host since they include all OpenType features in their served font (which we need here), but the mechanism is the same for using e.g.
```

Reply:
did you mean to suggest anything here?